astroplan/plots/core.py

```diff
@@ -69,19 +73,21 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None):
         style_kwargs = {}
     style_kwargs = dict(style_kwargs)
     style_kwargs.setdefault('linestyle', '-')
-    style_kwargs.setdefault('color', 'k')
     style_kwargs.setdefault('fmt', '-')
 
     # Populate time window if needed.
+    time = Time(time)
     if time.isscalar:
         time = time + np.linspace(-12, 12, 100)*u.hour
+    elif len(time) == 1:
+        print("Danger: You have used a non-scalar Time object of length 1, and"
```

Comment:
This should be a warning rather than a print statement. @eteq is `astropy.log.warning()` good enough?